### PR TITLE
Add addtag command

### LINKED ISSUE #79: New `addtag` command

### PR DESCRIPTION
**Pull request by Arixeyeion**
The addtag command allows users to add tags without the need to do it by editing, multiple tags can be added at once.

Closes #79

### CODE REVIEW

File: src/main/java/seedu/findvisor/logic/parser/AddtagCommandParser.java [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+        }
+
+        return new AddtagCommand(index, newTags);
+    }
+
+    /**
+     * Returns true if none of the prefixes contains empty {@code Optional} values in the given
+     * {@code ArgumentMultimap}.
+     */
+    private static boolean arePrefixesPresent(ArgumentMultimap argumentMultimap, Prefix... prefixes) {
```
[the remark this answers is not in the document]
Comment (Arixeyeion) [reply src/main/java/seedu/findvisor/logic/parser/AddtagCommandParser.java:48]: I agree with the comments above and will work to change them

File: src/test/java/seedu/findvisor/logic/commands/CommandTestUtil.java [hunk 2]
```diff
@@ -73,6 +82,12 @@ public class CommandTestUtil {
     public static final String PREAMBLE_WHITESPACE = "\t  \r  \n";
     public static final String PREAMBLE_NON_EMPTY = "NonEmptyPreamble";
 
+    public static final String[] ARRAY_OF_MULTIPLE_VALID_TAG_STRINGS = {VALID_TAG_FRIEND, VALID_TAG_FINANCIAL_PLAN};
+    public static final String[] ARRAY_OF_SINGLE_VALID_TAG_STRING = {VALID_TAG_FINANCIAL_PLAN};
+    public static final Set<Tag> SET_OF_VALID_TAGS = new SampleDataUtil().getTagSet(
+            ARRAY_OF_MULTIPLE_VALID_TAG_STRINGS);
+    public static final Set<Tag> SET_OF_VALID_TAG = new SampleDataUtil().getTagSet(
+            ARRAY_OF_SINGLE_VALID_TAG_STRING);
```
[the remark this answers is not in the document]
Comment (Arixeyeion) [reply src/test/java/seedu/findvisor/logic/commands/CommandTestUtil.java:90]: I agree, will change it

File: src/main/java/seedu/findvisor/logic/parser/AddTagCommandParser.java [hunk 3]
```diff
@@ -0,0 +1,51 @@
...
+        }
+
+        return new AddTagCommand(index, newTags);
+    }
+
+    /**
+     * Returns true if none of the prefixes contains empty {@code Optional} values in the given
+     * {@code ArgumentMultimap}.
+     */
+    private static boolean arePrefixesPresent(ArgumentMultimap argumentMultimap, Prefix... prefixes) {
```
[the remark this answers is not in the document]
Comment (Arixeyeion) [reply src/main/java/seedu/findvisor/logic/parser/AddTagCommandParser.java]: This command is removed in a later commit

File: src/main/java/seedu/findvisor/logic/commands/AddTagCommand.java [hunk 4]
```diff
@@ -0,0 +1,126 @@
...
+        if (other == this) {
+            return true;
+        }
+
+        // instanceof handles nulls
+        if (!(other instanceof AddTagCommand)) {
+            return false;
+        }
+
+        AddTagCommand otherAddtagCommand = (AddTagCommand) other;
```
[the remark this answers is not in the document]
Comment (Arixeyeion) [reply src/main/java/seedu/findvisor/logic/commands/AddTagCommand.java]: I agree, will fix

**Review by KohGuanZeh — APPROVED** [review]
LGTM

**Review by Javiery3889 — APPROVED** [review]
LGTM

**Pull request merged**